pkg/controller/controller_test.go

```diff
@@ -1281,8 +1294,21 @@ func TestReconcileBindingNamespaceError(t *testing.T) {
 	testController.reconcileBinding(binding)
 
 	actions := filterActions(fakeCatalogClient.Actions())
-	if a := len(actions); a != 0 {
-		t.Fatalf("Unexpected number of actions: expected 0, got %v. Actions: %+v", a, actions)
+	if a := len(actions); a != 1 {
+		t.Fatalf("Unexpected number of actions: expected 1, got %v. Actions: %+v", a, actions)
+	}
+
+	var (
```

Comment:
Just curious why this var block is needed here?
Why not just change L1305 below to:
updateAction := actions[0].(clientgotesting.UpdateAction)
and
L1309 to:
updatedBinding := updateAction.GetObject().(*v1alpha1.Binding)